AnnoyancesFilter/sections/popups.txt

```diff
@@ -4749,9 +4749,9 @@ wegame.com.ua#$#.modal-subscribe { display: none!important; }
 wegame.com.ua#$#.modal-backdrop { display: none!important; }
 wegame.com.ua#$#body { overflow: visible!important; }
 !
-texterra.ru#$#body > div.mfp-wrap { display: none!important; }
-texterra.ru#$#body > div.mfp-bg { display: none!important; }
-texterra.ru#$#body { overflow: visible!important; }
+udostepnijto.pl,texterra.ru#$#body > div.mfp-wrap { display: none!important; }
+udostepnijto.pl,texterra.ru#$#body > div.mfp-bg { display: none!important; }
+udostepnijto.pl,texterra.ru#$#body { overflow: visible!important; }
```

Comment:
It seems that it's ad leftover, so please add rules to `foreign.txt`, for example here:
https://github.com/AdguardTeam/AdguardFilters/blob/4898beeaace0e77d9a65e4e68e91c31f914d25cd/EnglishFilter/sections/foreign.txt#L6114-L6115

Please also add a rule for `html` tag instead of `body` and change `overflow: visible` to `overflow: auto`, because there is additional scrollbar on the bottom if `visible` is used (at least on my side).

```
udostepnijto.pl#$#body > div.mfp-wrap { display: none !important; }
udostepnijto.pl#$#body > div.mfp-bg { display: none !important; }
udostepnijto.pl#$#html { overflow: auto !important; margin-right: 0 !important; }
```